idf_component.yml

```diff
@@ -0,0 +1,4 @@
+version: "2.0.0"
+description: "Drive parallel e-Paper displays with epdiy-based boards."
+url: "https://github.com/vroland/epdiy"
+license: LGPLv3
```

Comment:
Can you please check the comment I made in the PR?
In the Espressif docs mentions: license: The license of the component. It should be a valid SPDX license identifier listed in https://spdx.org/licenses/ This field is optional. The license will be automatically detected if it’s not specified.
LGPLv3 does not seem to be listed in SPDX so leaving this field as is will drop a warning on any user trying to use IDF component manager

Reply:
should be fixed now.